2022-05-metafacture-workshop/index.html

```diff
@@ -0,0 +1,944 @@
+	},
+	'001A' {
+		'0': '1140:08-12-99'
+	},
+	'001B' {
+		'0': '9999:05-01-08',
+		't': '22:57:29.000'
+	},
+	'001D' {
+		'0': '9999:99-99-99'
+	},
+	'001U' {
+		'0': 'utf8'
+	},
+	'001X' {
+		'0': '0'
+	},
+	'002@' {
+		'0': 'Aa'
+	},
+	'003@' {
+		'0': '958090564'
+	},
+	'004A' {
+		'f': 'kart. : DM 9.70, EUR 4.94, sfr 8.00, S 68.00'
+	},
+	'006U' {
+		'0': '00,B05,0285'
+	},
+	'007I' {
+		'S': 'o',
+		'0': '76088278'
+	},
+	'011@' {
+		'a': '1999'
+	},
+	'017A' {
+		'a': 'rb',
+		'a': 'si'
+	},
+	'019@' {
+		'a': 'XA-AT'
+	},
+	'021A' {
+		'a': 'Zukunft Bildung',
+		'h': 'Politische Akademie. [Hrsg. von Günther R. Burkert-Dottolo und Bernhard Moser]'
+	},
+	'028C' {
+		'9': '130681849',
+		'7': 'Tp1',
+		'V': 'piz',
+		'A': 'gnd',
+		'0': '130681849',
+		'E': '1952',
+		'a': 'Burkert',
+		'd': 'Günther R.',
+		'B': 'Hrsg.'
+	},
+	'033A' {
+		'p': 'Wien',
+		'n': 'Polit. Akad.'
+	},
+	'034D' {
+		'a': '79 S.'
+	},
+	'034I' {
+		'a': '24 cm'
+	},
+	'036F' {
+		'x': '299 12',
+		'9': '551720077',
+		'g': 'Adn',
+		'7': 'Tb1',
+		'A': 'gnd',
+		'0': '1040469-7',
+		'a': 'Politische Akademie',
+		'g': 'Wien',
+		'Y': 'PA-Information',
+		'h': 'Politische Akademie, WB',
+		'p': 'Wien',
+		'J': 'Politische Akad., WB',
+		'l': '99,2'
+	},
+	'036F/01' {
+		'x': '12',
+		'9': '025841467',
+		'g': 'Advz',
+		'i': '2142105-5',
+		'Y': 'Aktuelle Fragen der Politik',
+		'h': 'Politische Akademie',
+		'p': 'Wien',
+		'J': 'Politische Akad. der ÖVP',
+		'l': 'Bd. 2'
+	},
+	'045E' {
+		'a': '22',
+		'd': '18',
+		'm': '370'
+	},
+	'047A' {
+		'S': 'FE',
+		'a': 'ta'
+	}
+}
+					</code></pre>
+				<p><small><a href="https://github.com/hbz/metafacture-flux-examples/tree/master/sample1">https://github.com/hbz/metafacture-flux-examples/tree/master/sample1</a></small></p>
+			</section>
+
+			<section>
+				<h2>Transformationsmodul</h2>
+				<p><span class="fragment">Manipulation von Feldnamen und -werten; filtern, kombinieren, normalisieren
+						etc.</span></p>
+				<p><span class="fragment">Feldwerte aus Lookup-Tabellen in externen Dateien (z.B. Freitextfelder ->
+						kontrollierte Vokabulare)</span></p>
+				<p><span class="fragment">Morph: XML-basiert, Feld- / Metadaten-Event-Ebene</span></p>
+				<p><span class="fragment">Fix: eigene, Catmandu-Fix-artige Sprache, Record-basiert</span></p>
+			</section>
+
+			<section>
+				<h2>Metafacture Fix: Ziele</h2>
+				<p><span class="fragment">Erleichterung der Transformationskonfiguration</span></p>
+				<p><span class="fragment">Anknüpfung an existierende Konfigurationssprache aus <a
+							href="https://github.com/LibreCat/Catmandu">Catmandu</a> (mittelfristiges Ziel:
+						Standardisierung, s. <a href="https://github.com/elag/FIG">https://github.com/elag/FIG</a>)</span></p>
+				<p><span class="fragment">Vergrößerung der Zielgruppe um Bibliothekar:innen und andere
+						Metadatenfachleute (bei uns z.B. in OERSI, erster Anwendungsfall und Entwicklungsbegleitung zu Fix)</span></p>
+			</section>
+
```

Comment:
Brauchen wir eine Seite für FIX-Pfade? sowas?

```suggestion
		<section>
				<h2>Metafacture Fix: Pfade</h2>
				<p><span class="fragment">Um die verschiedenen Elemente und Felder für die Transformation anzuwählen, muss man ihre für FIX spezifischen Pfade angeben</span></p>
				<p><span class="fragment">
				einfache Elemente der obersten Ebene: `id`
				Elemente auf einer unteren Ebene: `title.subtitle`
				Arrays/wiederholende Felder, werden als Listen mit Index-Nummer angageben:
				`creator.1.name.firstName` (sich wiederholendes Feld)
				`creator[].3.name.firstName` (genuine arrays mit [])
				
				</span></p>
			</section>
```